### PR TITLE
- Keycloak added response codes and messages for error

### PR DESCRIPTION
**Pull request by Sri-Harsha06**
- Grouped config(.yml) files under resources folder
- Removed few logs
-  Deleted unwanted files from the generated application

### CODE REVIEW

File: generator-jhipster-gomicro/generators/server/generator.mjs [hunk 1]
```diff
@@ -11,11 +11,7 @@ export default class extends ServerGenerator {
     if (this.options.help) return;
 
     if (!this.options.jhipsterContext) {
-      throw new Error(
-        `This is a JHipster blueprint and should be used only like ${chalk.yellow(
-          "jhipster --blueprints test"
-        )}`
-      );
+      throw new Error(`This is a JHipster blueprint and should be used only like ${chalk.yellow('jhipster --blueprints test')}`);
```
Comment (craxkumar) [inline generator-jhipster-gomicro/generators/server/generator.mjs]: jhipster --blueprints test

change this to 
jhipster --blueprints gomicro

File: generator-jhipster-gomicro/generators/server/templates/gomicro/gomicro/auth/auth.go [hunk 2]
```diff
@@ -46,11 +46,15 @@ func Protect(next http.Handler) http.Handler {
...
 			logger.Errorf("Inspection failed: %s", err.Error())
+			w.WriteHeader(401)
+			json.NewEncoder(w).Encode(err.Error())
 			return
 		}
 		istokenvalid := *rptResult.Active
 		if !istokenvalid {
 			logger.Errorf("Token is not active")
+			w.WriteHeader(401)
+			json.NewEncoder(w).Encode("Token is not active")
```
Comment (craxkumar) [inline generator-jhipster-gomicro/generators/server/templates/gomicro/gomicro/auth/auth.go]: u can simply say:-

 Invalid token
or 
token expired

File: generator-jhipster-gomicro/generators/server/templates/gomicro/gomicro/eurekaregistry/helper/HttpHelper.go [hunk 3]
```diff
@@ -15,8 +14,6 @@ type OperationBody interface {
 
 // MakePostCall
 func MakePostCall(urlToPost string, body OperationBody, headers map[string]string) (error, *http.Response) {
-	msg := fmt.Sprintf("In MakePostCall to %s with body %s", urlToPost, body)
-	logger.Infof(msg)
```
Comment (craxkumar) [inline generator-jhipster-gomicro/generators/server/templates/gomicro/gomicro/eurekaregistry/helper/HttpHelper.go]: why remove a log for the POST call for eureka ?

File: generator-jhipster-gomicro/generators/server/templates/gomicro/gomicro/eurekaregistry/helper/HttpHelper.go [hunk 4]
```diff
@@ -46,13 +42,10 @@ func MakePostCall(urlToPost string, body OperationBody, headers map[string]strin
 		logger.Errorf("Error while making post call " + err.Error())
 		return errors.New("Error while making post call " + err.Error()), nil
 	}
-	logger.Infof("Successfull POST call with HTTP Status : " + resp.Status)
 	return nil, resp
 }
 
 func MakePutCall(urlToPost string, body OperationBody, headers map[string]string) (error, *http.Response) {
-	msg := fmt.Sprintf("In MakePostCall to %s with body %s", urlToPost, body)
```
Comment (craxkumar) [inline generator-jhipster-gomicro/generators/server/templates/gomicro/gomicro/eurekaregistry/helper/HttpHelper.go:54]: why remove a log for the PUT call for eureka ?
in spring boot application provided by the Jhipster we do have logs for it !

**Review by craxkumar — CHANGES_REQUESTED** [review]

File: generator-jhipster-gomicro/generators/server/generator.mjs [hunk 5]
```diff
@@ -278,6 +157,134 @@ export default class extends ServerGenerator {
...
+        };
+
+        const templatePaths = [
+          { src: 'docker', dest: 'docker' },
+          { src: 'proto', dest: 'proto' },
+          { src: 'go.mod', dest: 'go.mod' },
+          { src: 'main.go', dest: 'main.go' },
+          { src: 'Dockerfile', dest: 'Dockerfile' },
+          { src: 'Makefile', dest: 'Makefile' },
+          { src: 'ReadMe.md', dest: 'ReadMe.md' },
```
Comment (rohithteja-cmi) [inline generator-jhipster-gomicro/generators/server/generator.mjs]: README.md

File: generator-jhipster-gomicro/package-lock.json
Comment (rohithteja-cmi) [inline generator-jhipster-gomicro/package-lock.json:1]: please remove this file and add in git ignore as well.

**Review by craxkumar — APPROVED** [review]
Review OK!

**Pull request merged**